CMakeLists.txt

```diff
@@ -813,6 +813,7 @@ if (USE_TORCH)
       "-DCMAKE_CXX_FLAGS=-isystem ${CMAKE_BINARY_DIR}/pytorch/src/pytorch/ -isystem ${SPDLOG_INCLUDE_DIR} -isystem ${PROTOBUF_INCLUDE_DIR}"
       "-DCMAKE_CUDA_FLAGS=-isystem ${CMAKE_BINARY_DIR}/pytorch/src/pytorch/ -isystem ${SPDLOG_INCLUDE_DIR} -isystem ${PROTOBUF_INCLUDE_DIR}"
       -DCMAKE_CUDA_COMPILER=/usr/local/cuda/bin/nvcc
+    BUILD_COMMAND ""
```

Comment:
This was accidentally commited, but it prevents pytorch_vision to rebuild twice, so we can let it there? (the library is built at install step anyway)

Reply:
ok for me